src/application/ApplicationDetail/ApplicationDetail.tsx

```diff
@@ -79,15 +87,27 @@ export class ApplicationDetail extends Component<Props, State> {
                   this.props.app!.name
                 }: Variants`}</BreadcrumbItem>
               </Breadcrumb>
-              <Text
-                component={TextVariants.h1}
-                style={{
-                  paddingTop: 40,
-                  paddingBottom: 20,
-                }}
-              >
-                {this.props.app!.name}
-              </Text>
+              <Split style={{ paddingTop: 20, paddingBottom: 10 }}>
+                <SplitItem>
+                  <Text component={TextVariants.h1}>
+                    {this.props.app!.name}
+                  </Text>
+                </SplitItem>
+                <SplitItem isFilled />
+                <SplitItem>
+                  <Button
+                    className={'button-small'}
+                    icon={<ShareIcon />}
+                    variant={ButtonVariant.primary}
+                    onClick={() =>
+                      this.setState({ sendNotificationModel: true })
+                    }
+                  >
+                    Send Notification To This App
+                  </Button>
```

Comment:
This button should be disabled if there are no variants.
Probably should be disabled even if we don't have any installation.

Reply:
It should b for no devicese, but it makes for easier testing if it isn't. 
Agree on the variant question